Sources/Purchasing/StoreKitAbstractions/StoreKitWorkarounds.swift

```diff
@@ -119,3 +119,25 @@ extension ReceiptFetcher {
     }
 
 }
+
+extension SKPaymentQueue {
+
+    @available(iOS 14.0, *)
+    @available(macOS, unavailable)
+    @available(tvOS, unavailable)
+    @available(watchOS, unavailable)
+    @available(macCatalyst, unavailable)
+    func presentCodeRedemptionSheetIfAvailable() {
```

Comment:
Should we add a couple tests for this?

Reply:
We would have to test `MockPaymentQueue` as a way to test that the method is called, but it feels silly to test a mock :/